chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -243,6 +246,20 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
                 if (props.controlProps?.hideStartChatButton) {
                     dispatch({ type: LiveChatWidgetActionType.SET_MINIMIZED, payload: event?.payload?.isChatHidden });
                 }
+                const dateNow = Date.now();
+                if (dateNow - lastLWICheckTimeRef.current > Constants.LWICheckOnVisibilityTimeout) {
+                    chatSDK?.getConversationDetails()
```

Comment:
Placeholdder for the getConversationDetails change

Reply:
Fixed as discussed